Templates/default/ExampleTemplate.stencil

```diff
@@ -1,12 +1,15 @@
 import ProjectDescription
 let nameAttribute: Template.Attribute = .required("name")
+let platformAttribute: Template.Attribute = .optional("platform", default: "iOS")
```

Comment:
There is no need to declare an extra variable, I'd move `.optional(...)` to `attributes` to make it a little bit simpler.